awx/settings/defaults.py

```diff
@@ -316,6 +328,11 @@ def IS_TESTING(argv=None):
     'django.contrib.auth.backends.ModelBackend',
 )
 
+# Django OAuth Toolkit settings
+OAUTH2_PROVIDER_APPLICATION_MODEL = 'oauth2_provider.Application'
+OAUTH2_PROVIDER_ACCESS_TOKEN_MODEL = 'oauth2_provider.AccessToken'
+OAUTH2_PROVIDER = {}
```

Comment:
May need to define `SCOPES` here. I would see scopes corresponding largely to our existing roles -- admin, execute, use, etc. in addition to only read/write, giving a user the ability to authorize an external app/site to act on their behalf, with restrictions.